resources/lang/pt_BR/messages.php

```diff
@@ -4,7 +4,7 @@
 	'max_in_the_zip_file' => 'O arquivo zip deve ter no máximo :quantity arquivo(s).',
 	'not_empty_zip' => 'O arquivo zip não pode está vazio.',
 	'zip_has_xml_file' => 'O arquivo zip deve conter um XML.',
-	'cnes_xml_identification' => 'XML com formato inválido. Por favor, verifique o código do IBGE, o campo de ORIGEM e o campo de DESTINO.', // phpcs:ignore
+	'cnes_xml_identification' => 'XML com formato inválido. Por favor, verifique o código do IBGE, o campo de ORIGEM, o campo de DESTINO e a se a versão do XML compatível é a 2.1', // phpcs:ignore
```

Comment:
Como o valor da versão ta sendo passada por parâmetro, acho que o melhor é alterar esse valor da mensagem de acordo com o valor passado para regra. A mensagem poderia ficar como na sugestão a abaixo, porém tem que alterar o `CnesXMLIdentification` para passar o valor no `message()`, tem um exemplo de como passar no `CnesXMLDate`

```suggestion
	'cnes_xml_identification' => 'XML com formato inválido. Por favor, verifique o código do IBGE, o campo de ORIGEM, o campo de DESTINO e a se a versão do XML compatível é a :version_xsd', // phpcs:ignore
```